lib/upipe-av/ubuf_av.c

```diff
@@ -134,8 +134,10 @@ static void ubuf_av_free(struct ubuf *ubuf)
             free(buf[i]);
         free(buf);
     }
-    av_frame_unref(ubuf_av->frame);
-    av_frame_free(&ubuf_av->frame);
+    if (ubuf_av->frame) {
+        av_frame_unref(ubuf_av->frame);
```

Comment:
I believe this `av_frame_unref()` is not required since `av_frame_free()` already unrefs the frame?